ws-nextjs-app/cypress/e2e/specialFeatures/atiAnalytics/assertions/articleLiteSiteLink.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-relative-packages */
+import { interceptATIAnalyticsBeacons, COMPONENTS } from '../helpers';
+import {
+  assertATIComponentClickEvent,
+  assertATIComponentViewEvent,
+} from '../../../../../../cypress/e2e/specialFeatures/atiAnalytics/assertions';
```

Comment:
```suggestion
} from '#cypress/e2e/specialFeatures/atiAnalytics/assertions';
```

Does using the alias work instead of having to manually find how many levels up you need to go for the import?

Reply:
Heya, it causes an issue with module resolution.

Reply:
Will add an alias in though, good shout.

Reply:
Hmm, it might require a bit of investigating, will create a tech ticket for it.